pm_icecon/bt/params/a2l1c.py

```diff
@@ -1,12 +1,7 @@
-"""a2l1c.py: bootstrap parameters file.
+"""Bootstrap parameters for AMSR2 CETB product.
 
-Bootstrap parameters for use with AMSR2 from new CETB product,
-  i.e. either NSIDC-0763 or NSIDC-0630 version 2
-
-Parameters were originally provided by 'ret_parameters_amsru2.f', except:
-  - new wxlimt values were calculated by during Spring 2023 NSIDC
-    investigation of weather effects for the NOAA CDR version 5 product
-  - Note: Southern hemisphere wxlimt goes from 18.596 to 14.3
+Bootstrap parameters for use with AMSR2 from new CETB product, i.e. either
+NSIDC-0763 or NSIDC-0630 version 2.
```

Comment:
As a result of some of my comparisons of the 0630v1 vs 0630v2 products, I think we are going to need to use different algorithm coefficients for the different versions.  There are reasonably large TB differences between same-day data in some of the channels.

It's also possible that these coefficients will get improved with better calibrations even for 0630v1.

For now, we don't have anything better to use with 0763/0630v2 than the 0630v1 coefficients, but we might want to log a 'warning' that these coefficients are provisional.

Reply:
^^^Actually, that comment refers more to use in NISE or CETB-direct code.  The coefficients are *not* likely to change for the CDR -- which uses AU_12 data.

...unless, of course, the product gets a new version; say from P04/R04/B04 to ?04.1 or ?05 or something.

In fact...we probably want to indicate that these coefficients are specifically calibrated for "B04" files, and "are presumed to be valid for P04 and R04 files".  And...decreasingly confident for other "flavors" of AMSR2 data (eg 0630v1, 0763/0630v2; each of which could be GRD or SIR)

Reply:
I think these comments are specific to the AMSR2 AU_SI data, right? These params are for `a2l1c`.

Reply:
Ah, my mistake though - I indicated AMSR2 in the docstring!

Reply:
Ok, I think I was just generally confused since I haven't thought all that much about the A2L1C stuff. 

This is AMSR2 data from CETB. I think more details about this data and where to get it would be useful for the `pm_tb_data` repo.